fcl/gen/genie/ci_simulation_genie_bnb_icarus.fcl

```diff
@@ -0,0 +1,11 @@
+# file: ci_simulation_genie_bnb_icarus.fcl
+# author: chilgenb@fnal.gov
+# created: Nov 1 2021
+# description: wrapper around simulation_genie_icarus_bnb.fcl
+#              that enables IFDH access of the GENIE flux
+#              files instead of DIRECT, required to run on grid
+
+#include "simulation_genie_icarus_bnb.fcl"
+
+physics.producers.generator: @local::icarus_genie_BNB
```

Comment:
Isn't this line redundant?
```suggestion
```
I would expect this to be already set by `simulation_genie_icarus_bnb.fcl`.

Reply:
Indeed it is!